src/styles.scss

```diff
@@ -873,10 +873,10 @@ $green-font-color: #19B028;
 .change-account-selector_list {
 	background-color: var(--secondary);
 	padding: 0px 0px 10px;
-	margin-top: 15px;
+	margin-top: 10px;
 	width: 100%;
-	margin-left: -4px;
-	z-index: 11;
+	margin-left: -10px;
+	z-index: 99;
 	border: 1px solid var(--border);
```

Comment:
Also, `margin-left: -10px;` causes this behavior for me both on chrome and firefox: https://imgur.com/a/8TWyFie
If you change it to `margin-left: 0px;` and remove `border` prop it looks like this: https://imgur.com/a/VmkN0la
What do you think?

Reply:
I think #286 should resolve the problem. We should have been using a drop down directive long ago. I've tested that PR on a number of browsers, does it solve the issue you're having?

Reply:
Yup, it works now. The only thing is, I feel like the dropdown became narrower. This causes the "blue dot" and the last 's' in "Send Creator Coins" to extend horizontally beyond the dropdown. Check it here: https://imgur.com/a/oUEYZKL
Maybe it could be a couple px wider?

Reply:
> I think #286 should resolve the problem. We should have been using a drop down directive long ago. I've tested that PR on a number of browsers, does it solve the issue you're having?  

Everything is fine as per my PR just have to change z-index:11 to z-index:99